### PR TITLE
update RW612 powerlimit table

### PR DESCRIPTION
**Pull request by MaochenWang1**
update RW612 powerlimit table

### CODE REVIEW

File: mcux/middleware/wifi_nxp/example/common/usb/usb_host_msd/ffconf.h [hunk 1]
```diff
@@ -0,0 +1,285 @@
+#ifndef _FFCONF_H_
```
Comment (dleach02) [inline mcux/middleware/wifi_nxp/example/common/usb/usb_host_msd/ffconf.h:1]: Let's not have any example code in the HAL

File: mcux/middleware/wifi_nxp/internal/doxygen/DoxygenLayout.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
+<doxygenlayout version="1.0">
+  <!-- Navigation index tabs for HTML output -->
```
Comment (dleach02) [inline mcux/middleware/wifi_nxp/internal/doxygen/DoxygenLayout.xml:2]: remove documentation from the HAL

File: mcux/middleware/wifi_nxp/internal/scripts/pick/bin/allMacros_iw61x.txt [hunk 3]
```diff
@@ -0,0 +1,134 @@
+CONFIG_MLAN_WMSDK
```
Comment (dleach02) [inline mcux/middleware/wifi_nxp/internal/scripts/pick/bin/allMacros_iw61x.txt:1]: remove this

File: mcux/middleware/wifi_nxp/internal/DPP Test  Instruction_v1.1.docx
Comment (dleach02) [inline mcux/middleware/wifi_nxp/internal/DPP Test  Instruction_v1.1.docx:1]: Remove the internal folder completely

Comment (MaochenWang1) [reply mcux/middleware/wifi_nxp/internal/DPP Test  Instruction_v1.1.docx:1]: all updated, thanks, and will hold this PR

**Comment by MaochenWang1** [comment]
Hi @dleach02 @mmahadevan108 
please review

**Pull request closed without merging**